src/Sylius/Behat/Context/Ui/Shop/ProductContext.php

```diff
@@ -111,4 +141,53 @@ public function iShouldSeeTheProductAttributeWithValue($attributeName, $Attribut
             sprintf('Product should have attribute %s with value %s, but it does not.', $attributeName, $AttributeValue)
         );
     }
+
+    /**
+     * @Given /^I want to see products in (channel "([^"]*)")$/
```

Comment:
`CurrentChannelContext` sounds good! And the step e.g. `Given I am browsing "([^"]*)" channel`.